WooCommerce/Classes/ViewRelated/Dashboard/Settings/Settings/SettingsViewModel.swift

```diff
@@ -0,0 +1,350 @@
+import UIKit
+import Combine
```

Comment:
Nit: This import seems redundant, perhaps it can be removed?

Reply:
Nice catch. Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5393/commits/e51af98e8c90b9196d1f38140f5e0b9c78fc4d5d